PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Infrastructure/Data/HospitalDbContextFactory.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+using Microsoft.Extensions.Configuration;
+using System.IO;
+
+namespace PrivateHospitals.Data.Data
+{
+    public class HospitalDbContextFactory : IDesignTimeDbContextFactory<HospitalDbContext>
+    {
+        public HospitalDbContext CreateDbContext(string[] args)
+        {
+            // Створення конфігурації для отримання рядка підключення
+            IConfigurationRoot configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json")
+                .Build();
+
+            var builder = new DbContextOptionsBuilder<HospitalDbContext>();
+            var connectionString = configuration.GetConnectionString("DefaultConnection");
+
+            builder.UseNpgsql(connectionString);
+
+            return new HospitalDbContext(builder.Options);
+        }
+    }
```

Comment:
It's not needed. To run migration for your infra assembly:
1. Place DbContext and migrations in your Infrastructure assembly
2. Google how to create migrations when you have appsettings in 1 assembly and Context in another

Reply:
Fixed